caratekit.sh

```diff
@@ -70,6 +70,7 @@ IA=5
 endimessage=""
 ALTERNATIVE_TEST_DATA_IDS=( "1524929477" "1524947605" "1532022061" )
 SHORT_TEST_DATA_IDS=( "1532022061_subset" )
+SPHEVERSION="1.1.0"
```

Comment:
Are we versioning @SpheMakh now? Just curious.

Reply:
we totally should!

Reply:
@SpheMakh is improving all the time, so yes, of course.